app/views/comments/_comment.html.erb

```diff
@@ -1,7 +1,21 @@
 <% if comment && comment.user %>
-  <% if comment.depth < 3 && !should_be_hidden?(comment, @root_comment) %>
-    <details open>
-      <summary><span>&nbsp;</span></summary>
+  <% if comment.depth < 4 && !should_be_hidden?(comment, @root_comment) %>
```

Comment:
did we add a new level of nesting?

Reply:
No, we are not. But I slightly changed the logic of displaying  and applying appropriate classes for child nodes (it used to be applied elsewhere, inside `details` element. So this should not really change the logic for how the comments are nested, just some mechanisms for how we display it..

I've tried to test it through and it worked okey on my local env but if you can think of a testing scenario for this and make sure it doesn't really change anything significant, I'd really appreciate your help.

Reply:
I just added a bunch of comments to the same thread and saw they don't get nested too much so I think we're okay :D

Reply:
FYI I reverted that logic to not make things too confusing :)